tests/acceptance/AuthorshipCest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Tests for core module's authorship features.
+ *
+ * phpcs:disable WordPress.Files, WordPress.NamingConventions, PSR1.Classes.ClassDeclaration.MissingNamespace, HM.Functions.NamespacedFunctions
+ */
+
+use Codeception\Util\Locator;
+
+/**
+ * Test core module admin features.
+ */
+class AuthorshipCest {
+
+	/**
+	 * Make sure Authorship is activated.
+	 *
+	 * @param AcceptanceTester $I Actor object.
+	 *
+	 * @return void
+	 */
+	public function _before( AcceptanceTester $I ) {
+		$I->bootstrapWith( [ __CLASS__, '_enableAuthorship' ] );
```

Comment:
Does this need rollback set to false? As I understand it the file will get cleaned up as soon as the headless browser navigates somewhere

Reply:
Nope, as soon as the test thread closes, not the acceptance test thread. I should actually rollback after the test.

Reply:
Ahhh of course 🤦‍♂️  - pretty complicated thinking between what's in the test thread and what's in the browser request thread! All LGTM :shipit: